content/docs/getting-started/overview/index.md

```diff
@@ -0,0 +1,56 @@
+- Dive into scorecards, templates and docs-as-code
+
+## 1. [Adding users](/docs/getting-started/user-management/)
+
+- [Invite](/docs/getting-started/user-management/) members of your team who will be responsible for Roadie. 
+- Designate [admins](/docs/getting-started/create-admin-group/) who will be responsible for configuring the Roadie system.
+- [Model users, groups and teams](/docs/getting-started/teams/) to build concepts of ownership into the catalog from the start.
+
+## 2. [Setting up your catalog](/docs/getting-started/adding-a-catalog-item/)
+
+We provide various ways to populate the catalog. Generally you will store entity files in a code repository and then provide Roadie with the URL of that file for it to be imported as a catalog entry. However you may also import a catalog item via the Roadie CLI or API.
+
+- **GitHub** - You can store a catalog item as yaml in a code repository on GitHub and import it into the Roadie catalog.
+- **Azure** - You can store a catalog item as yaml in a code repository in Azure and import it into the Roadie catalog.
+- **BitBucket** - You can store a catalog item as yaml in a code repository in BitBucket and import it into the Roadie catalog.
+- **AWS S3** - You can store a catalog item as yaml in an S3 bucket in AWS and import it into the Roadie catalog.
+- **Roadie CLI** - You can import a catalog item into Roadie by calling it's API via the Roadie CLI.
+- **Roadie API** - You can import a catalog item into Roadie by calling it's API.
+
+## 3. [Modelling your software](/docs/getting-started/model-software/)
+
+Creating a structure for your software catalog allows you to establish and keep order in the way in which your software is developed.
+
+## 4. [Installing plugins](/docs/getting-started/configuring-backstage-plugins/)
+
+Roadie supports many open-source Backstage plugins out of the box. See the[Plugins & Integrations](/docs/integrations/) page for a full list of supported plugins, and dedicated instructions for each one. The list of available plugins is also visible to admins in the Administration area.
+
+## 5. [Configuring the Roadie UI](/docs/getting-started/configure-ui/)
+
+Roadie allows you to configure which plugins you want to use via our UI. This guide explains the different types of plugin components and the different places they can be integrated into the app.
+
+## 6. [Creating a scorecard](/docs/getting-started/creating-a-scorecard/)
+
+Roadie Tech Insights helps you keep track of all of your software assets and make sure they meet your quality and compliance targets. You can create Scorecards to track the things that matter about your assets in the Backstage catalog. 
+
+## 7. [Creating a template](/docs/getting-started/scaffolding-components/)
+
+The Roadie Backstage Scaffolder is a feature that allows you to define software templates to create new software projects, update existing ones or simply perform repeated tasks in a consistent manner.
```

Comment:
This start deviates from the rest of the sections, propose dropping `The` from the sentence.